src/main/java/org/apache/commons/compress/archivers/cpio/CpioArchiveInputStream.java

```diff
@@ -406,7 +406,11 @@ public int read(final byte[] b, final int off, final int len)
     private long readAsciiLong(final int length, final int radix)
             throws IOException {
         final byte[] tmpBuffer = readRange(length);
-        return Long.parseLong(ArchiveUtils.toAsciiString(tmpBuffer), radix);
+        try {
+            return Long.parseLong(ArchiveUtils.toAsciiString(tmpBuffer), radix);
+        } catch(NumberFormatException ex) {
```

Comment:
Fix formatting

Reply:
You need a space after "catch" and before "(". I'll have to fix our checkstyle configuration...